test/addons-napi/test_env_sharing/test.js

```diff
@@ -5,6 +5,6 @@ const storeEnv = require(`./build/${common.buildType}/store_env`);
 const compareEnv = require(`./build/${common.buildType}/compare_env`);
 const assert = require('assert');
 
-assert.strictEqual(compareEnv(storeEnv), true,
-                   'N-API environment pointers in two different modules have ' +
-                   'the same value');
+// N-API environment pointers in two different modules have the same value
+
```

Comment:
```suggestion
```
nit: line no. 9 can be removed, as the comment is related to `assert.strictEqual`